tests/test_eval.py

```diff
@@ -41,24 +44,30 @@ def get_quantized_marian_args(data_dir: DataDir, model_name: str):
     ]  # fmt: skip
 
 
+comet_forwards = 0.3268
+comet_backwards = 0.4306
+
 test_data = [
     # task_name                                                   model_type   model_name
-    ("evaluate-backward-sacrebleu-wmt09-en-ru",                   "base",      "final.model.npz.best-chrf.npz"),
-    ("evaluate-finetuned-student-sacrebleu-wmt09-en-ru",          "base",      "final.model.npz.best-chrf.npz"),
-    ("evaluate-student-sacrebleu-wmt09-en-ru",                    "base",      "final.model.npz.best-chrf.npz"),
-    ("evaluate-teacher-ensemble-sacrebleu-sacrebleu_wmt09-en-ru", "base",      "model*/*.npz"),
-    ("evaluate-quantized-sacrebleu-wmt09-en-ru",                  "quantized", "model.intgemm.alphas.bin")
+    ("evaluate-backward-sacrebleu-wmt09-en-ru",                   "base",      "final.model.npz.best-chrf.npz", comet_backwards),
+    ("evaluate-finetuned-student-sacrebleu-wmt09-en-ru",          "base",      "final.model.npz.best-chrf.npz", comet_forwards),
+    ("evaluate-student-sacrebleu-wmt09-en-ru",                    "base",      "final.model.npz.best-chrf.npz", comet_forwards),
+    ("evaluate-teacher-ensemble-sacrebleu-sacrebleu_wmt09-en-ru", "base",      "model*/*.npz",                  comet_forwards),
+    ("evaluate-quantized-sacrebleu-wmt09-en-ru",                  "quantized", "model.intgemm.alphas.bin",      comet_forwards)
 ]  # fmt:skip
 
 
 @pytest.mark.parametrize("params", test_data, ids=[d[0] for d in test_data])
 def test_evaluate(params) -> None:
-    (task_name, model_type, model_name) = params
+    (task_name, model_type, model_name, comet) = params
```

Comment:
should we separate running comet to another test so that we at least can easily run the fast ones?

Reply:
If we do maybe we should follow-up. I'm a bit annoyed how long this takes to run. I spent a bit of time trying to train my own comet model that was much smaller so that it could potentially run faster.

Splitting up the test would be hard since it's a run_task and not a unit test where it's easy to run just parts of it. I'd have to add a bunch of environment variables that artificially not do work, and then the assertions would get a bit messy to run.

Reply:
Of course I figured it out after typing this. I split up the test.